src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -96,44 +92,43 @@ static auto logger = ert::get_logger("job_queue.lsf_driver");
 #define DEFAULT_BHIST_CMD "bhist"
 
 struct lsf_job_struct {
-    long int lsf_jobnr;
+    long int lsf_jobnr = 0;
     /** Used to look up the job status in the bjobs_cache hash table */
-    char *lsf_jobnr_char;
-    char *job_name;
+    char *lsf_jobnr_char = nullptr;
+    char *job_name = nullptr;
 };
 
 struct lsf_driver_struct {
-    char *queue_name;
-    char *resource_request;
-    std::vector<std::string> exclude_hosts;
-    char *login_shell;
-    char *project_code;
-    pthread_mutex_t submit_lock;
-
-    lsf_submit_method_enum submit_method;
-    int submit_sleep;
-
-    int error_count;
-    int submit_error_sleep;
+    char *queue_name = nullptr;
+    char *resource_request = nullptr;
+    std::vector<std::string> exclude_hosts{};
+    char *login_shell = nullptr;
+    char *project_code = nullptr;
+    pthread_mutex_t submit_lock{};
+
+    lsf_submit_method_enum submit_method = LSF_SUBMIT_LOCAL_SHELL;
+    int submit_sleep = 0;
+    int error_count = 0;
+    int submit_error_sleep = SUBMIT_ERROR_SLEEP * 1000000;
 
     /*-----------------------------------------------------------------*/
     /* Fields used by the shell based functions */
-    bool debug_output;
-    int bjobs_refresh_interval;
-    time_t last_bjobs_update;
+    bool debug_output = false;
+    int bjobs_refresh_interval = 0;
+    time_t last_bjobs_update = time(nullptr);
```

Comment:
I've kept the reinitialization of this variable, since I am concerned that this happens compile-time, and not run-time. Timing might be affected if this is not updated.

```
static void lsf_driver_shell_init(lsf_driver_type *lsf_driver) {
     lsf_driver->last_bjobs_update = time(nullptr);
     lsf_driver->bjobs_cache = hash_alloc();
     lsf_driver->my_jobs = hash_alloc();
 }
```